packages/dashboard-hardhat-plugin/package.json

```diff
@@ -0,0 +1,55 @@
+{
+  "name": "@truffle/dashboard-hardhat-plugin",
+  "version": "0.1.0-0",
+  "description": "Truffle Dashboard for Hardhat plugin",
+  "repository": "github:trufflesuite/dashboard-hardhat-plugin",
+  "author": "ConsenSys Software Inc",
+  "license": "MIT",
+  "main": "dist/src/index.js",
+  "types": "dist/src/index.d.ts",
+  "keywords": [
+    "ethereum",
+    "smart-contracts",
+    "hardhat",
+    "hardhat-plugin",
+    "truffle",
+    "truffle-dashboard"
+  ],
+  "scripts": {
```

Comment:
```suggestion
  "scripts": {
    "prepare": "yarn build",
```

Reply:
Removed in a separate commit so I'll resolve without committing this one